packages/extension-commerce-bm-seo/config/default.json

```diff
@@ -1,5 +1,6 @@
 {
     "enabled": true,
+    "matchingStrategy": "API_FIRST",
```

Comment:
I know that the name "matchingStrategy" came from our discussion and the inspo from service worker fetching strategies, so I'll take blame for that.... but I think this name isn't as self explanatory as it could be. 

Can you ping the broader team with a few more options and get some feedback on potential other names for this option.